lib/class-wp-theme-json-gutenberg.php

```diff
@@ -1307,7 +1307,13 @@ private static function remove_insecure_settings( $input ) {
 					esc_attr( esc_html( $preset['name'] ) ) === $preset['name'] &&
 					sanitize_html_class( $preset['slug'] ) === $preset['slug']
 				) {
-					$value           = $preset[ $preset_metadata['value_key'] ];
+					$value = null;
+					if ( isset( $preset_metadata['value_key'] ) ) {
+						$value = $preset[ $preset_metadata['value_key'] ];
+					} elseif ( isset( $preset_metadata['value_func'] ) ) {
+						$value = call_user_func( $preset_metadata['value_func'], $preset );
+					}
```

Comment:
It would be good for the way we check `value_key` and `value_func` to be consistent with how its done in [`get_settings_values_by_slug`](https://github.com/WordPress/gutenberg/pull/35255/files#diff-b03597cc3da199e5aa5a94950e8241135904853e7c3b82d758e42744878afae7R696-R706).

I don't have much of an opinion on which way is better—just that they behave the same.